modules/aws-private-link-cluster/variables.tf

```
@@ -0,0 +1,21 @@
+variable "project_id" {
+  description = "The project id (e.g., 65def6ce0f722a1507105bb5)."
+  type        = string
+}
+
+variable "vpc_id" {
+  description = "The ID of the VPC in which the endpoint will be used."
```

```suggestion
  description = "The ID of the VPC for the endpoint to use."
```
Suggestion: Remove passive voice.